### PR TITLE
dev-util/bitrise: bump version 1.20.0

### PR DESCRIPTION
**Pull request by koral--**

### CODE REVIEW

File: dev-util/bitrise/bitrise-1.20.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+src_test() {
+	export PULL_REQUEST_ID=""
+	export INTEGRATION_TEST_BINARY_PATH="${S}/bin/bitrise"
+	GOPATH="${WORKDIR}" go test -v ./_tests/integration/... || die "Integration tests failed"
+}
+
+src_install() {
+	dobin bin/bitrise
+	dodoc README.md
+	use doc && dodoc -r _docs
```
[the remark this answers is not in the document]
Comment (koral--) [reply dev-util/bitrise/bitrise-1.20.0.ebuild]: Yes, I follow original folder structure here. Additionally `README.md` file exists both in `_docs` and root directory.
Do you recommend another solution for such case?

**Comment by mgorny** [comment]
I get a lots of test failures, apparently due to network-sandbox. It is forbidden for tests to use Internet. Could you verify whether they can be done to work locally? (they apparently try to clone some plugin or sth)

[dev-util:bitrise-1.20.0:20180809-073436.log](https://github.com/gentoo/gentoo/files/2273232/dev-util.bitrise-1.20.0.20180809-073436.log)

**Comment by koral--** [comment]
Bitrise is tightly coupled with GitHub. It seems that the only reasonable way is to skip tests requiring network connection.
Thanks for pointing out that it is forbidden to use network in tests. I've read this guide: https://wiki.gentoo.org/wiki/Package_testing but it is not explicitly mentioned there.

I'll figure out which tests require network and exclude them.

**Comment by mgorny** [comment]
Thanks. FTR, some good tips can be found @ https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html

**Comment by koral--** [comment]
For now I've just removed tests failing without network. I'll talk with upstream whether they accept some more elegant solution e.g. Go build tags.

File: dev-util/bitrise/bitrise-1.20.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+src_compile() {
+	GOPATH="${WORKDIR}" go build -v -o bin/bitrise || die "Couldn't compile bitrise"
+}
+
+src_test() {
+	pushd "${S}/_tests/integration/" || die "Couldn't find integration tests directory"
+	rm envstore_test.go exit_code_test.go global_flag_test.go json_params_test.go log_filter_test.go\
+		 output_alias_test.go step_template_test.go trigger_params_test.go update_test.go timeout_test.go
```
Comment (mgorny) [inline dev-util/bitrise/bitrise-1.20.0.ebuild]: `|| die`. Also please add a space before backslash.

**Comment by mgorny** [comment]
Thanks.

**Pull request closed without merging**